### ISSUE TITLE
can't see heat map 

### ISSUE THREAD

**Issue by pavankat** [Qbatch/heatmap-rails#2]
the clicks and mouse movements get saved in the database but when I do 

```
<%= show_heatmap(request.path) %> 
```

nothing happens

in the js console I see this

```
Uncaught TypeError: Cannot read property 'getBoundingClientRect' of null
```

clicking in it's on this line:

```
    width = getElement(path.xpath).getBoundingClientRect().width;
```

**Comment by hassanakram** [comment]
@pavankat Can you please try out the latest version & see if the issue is resolved?

**Comment by jafuentest** [comment]
Just installed and I can confirm that, on Firefox the error is: `TypeError: getElement(...) is null`

**Comment by jafuentest** [comment]
Issue was partially fixed at #3 
I had some troubles with the scrolls and submitted #4 to fix it
Someone should bump the version and update on rubygems
